query/src/pipelines/new/processors/transforms/aggregator/aggregator_final.rs

```diff
@@ -105,9 +112,10 @@ impl<Method: HashMethod + PolymorphicKeysHelper<Method> + Send> FinalAggregator<
 
             match inserted {
                 true => {
-                    let place = state.alloc_layout2(params);
-                    places.push(place);
-                    entity.set_state_value(place.addr());
+                    if let Some(place) = state.alloc_layout2(params) {
```

Comment:
What happen with layout is None?

Reply:
It means there is no state and no memory address will be returned.